lib/components/tree.component.ts

```diff
@@ -10,7 +10,7 @@ import { KEYS } from '../constants/keys';
 import * as _ from 'lodash';
 
 @Component({
-  selector: 'Tree',
+  selector: 'Tree, tree',
```

Comment:
Let's just add a warning message in the constructor that says: "if you are using the UpperCase selector please move to the kebab-case selector, as the UpperCase will be soon deprecated".

Can you also add kebab-case selectors to all other directives and components of the library?

Thanks for the help!